ignite/metrics/mean_absolute_error.py

```diff
@@ -1,5 +1,7 @@
 from __future__ import division
```

Comment:
Same here.
Maybe we can remove above line too as no more python 2:
```python
from __future__ import division
```

Reply:
Will do this in a separate PR